sdk/identity/Azure.Identity/CHANGELOG.md

```diff
@@ -12,6 +12,10 @@ Thank you to our developer community members who helped to make Azure Identity b
 
 - Added `AzurePowerShellCredential` to `DefaultAzureCredential` (A community contribution, courtesy of _[olandese](https://github.com/olandese))_
 
+### Breaking Changes
```

Comment:
Do we also want to add Fixes and improvements to include logging improvements (for msal and managed identity discovery(, and this fix https://github.com/Azure/azure-sdk-for-net/pull/18315?